website/docs/reference/commands/list.md

```diff
@@ -72,7 +72,7 @@ $ dbt ls --select snowplow.* --output json
 
 **Listing JSON output with custom keys**
 ```
-$ dbt ls --select snowplow.* --output json --output-keys name description
+$ dbt ls --select snowplow.* --output json --output-keys name,description
```

Comment:
correcting this per @jtcohen6 's suggestion, however, it will eventually work in 1.5 _without_ quotes

```suggestion
$ dbt ls --output json --output-keys "name resource_type description"
```